### PR TITLE
Multistepform

### PR DESCRIPTION
**Pull request by manuelwecker**
Implemented flexible footer button to navigate through the form and pages
Bildschirmfoto 2020-01-10 um 11 09 27

### CODE REVIEW

File: client/src/components/Aside.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import styled from "@emotion/styled";
+
+const Aside = styled.div`
```
Comment (leonp5) [inline client/src/components/Aside.js]: Aside doesnt tell me really, what it does

File: client/src/components/Footer.js [hunk 2]
```diff
@@ -16,6 +16,16 @@ const Button = styled.button`
   color: ${props => props.theme.colors.secondary};
 `;
 
+const None = styled(Link)`
```
Comment (leonp5) [inline client/src/components/Footer.js]: isnt here missing a "e" for `none` ?

Comment (manuelwecker) [reply client/src/components/Footer.js]: Thx

File: client/src/components/Aside.js (hunk 1 above)
Comment (manuelwecker) [reply client/src/components/Aside.js]: Thx! It's an styling issue, I have already a semantic footer component

**Pull request merged**